src/lib/Commands/TestCommand/index.js

```diff
@@ -337,10 +388,18 @@ ${bootstrapCode}
 }
 
 // resolve modules
-__module_tests(__module_ImpUnit_Promise_exports, __module_impUnit_exports.ImpTestCase);
```

Comment:
If `isPending` is ever used in the text sources, then the tests will need to be updated. So it shouldn't be an issue.